packages/loader/container-loader/src/containerContext.ts

```diff
@@ -282,6 +282,25 @@ export class ContainerContext implements IContainerContext {
     }
 
     // #region private
+    private async initializeFromExisting() {
+        const runtimeFactory = await this.getRuntimeFactory();
+        this._runtime = await runtimeFactory.instantiateFromExisting(this);
```

Comment:
Back compat is missing. Tests should catch it - they should fail

Reply:
Can you expand on this, I'm a bit confused? Is it that we'd need to query the fact that the implementation we're getting has the method we're looking for and if not we fallback to the `instantiateRuntime` method? We can sync offline if needed.

Reply:
That's correct. I believe (though need to double check) that we test compat in both directions, i.e. old loader / new runtime &  new loader / old runtime. Later is less interesting, but support is missing here. And I think there is a chance customer runs into it with Bohemia due to caching, i.e. for a some small amount of time user might be getting latest loader but cached (old) container runtime.